commit e597ffe258d510defbd4eadef2b0b4d07b903b47 (refs/changes/52/3952/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-05-14T15:22:14-07:00 (1 year, 5 months ago)
    
    TRITON-385 cloudapi has a vestigial logadm.conf entry for /var/log/cloudapi.log
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index c9e2eab..0775d3f 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -160,7 +160,6 @@ sdc_log_rotation_add cloudapi-8081 /var/svc/log/*cloudapi-8081.log 1g
 sdc_log_rotation_add cloudapi-8082 /var/svc/log/*cloudapi-8082.log 1g
 sdc_log_rotation_add cloudapi-8083 /var/svc/log/*cloudapi-8083.log 1g
 sdc_log_rotation_add cloudapi-8084 /var/svc/log/*cloudapi-8084.log 1g
-sdc_log_rotation_add cloudapi /var/log/cloudapi.log 1g
 sdc_log_rotation_setup_end
 
 # All done, run boilerplate end-of-setup
